src/shared-components/theme-engine/theme.json

```diff
@@ -88,6 +95,9 @@
     "fontColorTertiary": "#959595",
     "fontColorTertiaryHover": "#000000",
 
+    "fontColorAppMenu": "#E6E6E6",
+    "filterAppMenuImage": "brightness(100%)",
```

Comment:
as with the above comment, i think filters, etc., don't need to be in the theme.